server/src/azrael/event.py

```diff
@@ -0,0 +1,191 @@
+    @classmethod
+    def from_doc(cls, doc):
+        """
+        Parses a mongodb document into a 'Lent' Azrael v1 event.
+
+        Args:
+            doc (Dict): a mongodb document
+
+        Returns:
+            _type_: instance of this class with the correct
+            configs.
+        """
+
+        tx_hash, tx_offset = Event.parse_id(doc['_id'])
+
+        return cls(
+            tx_hash,
+            tx_offset,
+            lendingId=int(doc['lendingId']), 
+            lentAmount=int(doc['lentAmount']), 
+            maxRentDuration=int(doc['maxRentDuration']),
+            paymentToken=int(doc['paymentToken']),
+            nftAddress=doc['nftAddress'],
+            tokenId=doc['tokenId'],
+            lendersAddress=doc['lendersAddress'],
+            dailyRentPrice=doc['dailyRentPrice'],
+            nftPrice=doc['nftPrice'],
+            isERC721=doc['isERC721'],
+            )
+
+
+@dataclass
+class RentedEvent(Event):
+    lendingId: int
+    renterAddress: str
+    rentDuration: int
+    rentedAt: int
+
+    @classmethod
+    def from_doc(cls, doc):
+        """
+        Parses a mongodb document into a 'Rented' Azrael v1 event.
+
+        Args:
+            doc (Dict): a mongodb document
+
+        Returns:
+            _type_: instance of this class with the correct
+            configs.
+        """
+
+        tx_hash, tx_offset = Event.parse_id(doc['_id'])
+
+        return cls(
+            tx_hash,
+            tx_offset,
+            lendingId=int(doc['lendingId']),
+            renterAddress=doc['renterAddress'],
+            rentDuration=int(doc['rentDuration']),
+            rentedAt=int(doc['rentedAt'])
+        )
+
+
+@dataclass
+class ReturnedEvent(Event):
+    lendingId: int
+    returnedAt: int
+
+    @classmethod
+    def from_doc(cls, doc):
+        """
+        Parses a mongodb document into a 'Returned' Azrael v1 event.
+
+        Args:
+            doc (Dict): a mongodb document
+
+        Returns:
+            _type_: instance of this class with the correct
+            configs.
+        """
+
+        tx_hash, tx_offset = Event.parse_id(doc['_id'])
+
+        return cls(
+            tx_hash,
+            tx_offset,
+            lendingId=int(doc['lendingId']),
+            returnedAt=int(doc['returnedAt'])
+        )
+
+@dataclass
+class LendingStopped(Event):
+    lendingId: int
+    stoppedAt: int
+
+    @classmethod
+    def from_doc(cls, doc):
+        """
+        Parses a mongodb document into a 'LendingStopped' Azrael v1 event.
+
+        Args:
+            doc (Dict): a mongodb document
+
+        Returns:
+            _type_: instance of this class with the correct
+            configs.
+        """
+
+        tx_hash, tx_offset = Event.parse_id(doc['_id'])
+
+        return cls(
+            tx_hash,
+            tx_offset,
+            lendingId=int(doc['lendingId']),
+            stoppedAt=int(doc['stoppedAt'])
+        )
+
+@dataclass
+class CollateralClaimed(Event):
+    lendingId: int
```

Comment:
snake case, please :) `lending_id` and `claimed_at`. Have a look if there are any other places that use camel case